File: src/main/java/com/koddy/server/coffeechat/domain/repository/query/spec/MentorCoffeeChatQueryCondition.java
package com.koddy.server.coffeechat.domain.repository.query.spec;

import com.koddy.server.coffeechat.domain.model.CoffeeChatStatus;

import java.util.List;

public record MentorCoffeeChatQueryCondition(
        long mentorId,
        List<CoffeeChatStatus> status
) {
}

File: src/main/java/com/koddy/server/coffeechat/domain/repository/query/spec/MentorCoffeeChatQueryCondition.kt
package com.koddy.server.coffeechat.domain.repository.query.spec

import com.koddy.server.coffeechat.domain.model.CoffeeChatStatus

data class MentorCoffeeChatQueryCondition(
    val mentorId: Long,
    val status: List<CoffeeChatStatus>,
)
